jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	preset: 'ts-jest',
```

Comment:
This is just a suggestion but [@swc-node/jest](https://npmjs.com/package/@swc-node/jest) can be a good alternative. It's very fast